Makefile

```diff
@@ -25,18 +31,34 @@ LEPTONICA_VERSION := 1.75.3
 # Tesseract commit. Default: $(TESSERACT_VERSION)
 TESSERACT_VERSION := fd492062d08a2f55001a639f2015b8524c7e9ad4
 
-# Tesseract model repo to use. Default: $(TESSDATA_REPO)
-TESSDATA_REPO = _best
-
 # Ground truth directory. Default: $(GROUND_TRUTH_DIR)
 GROUND_TRUTH_DIR := data/ground-truth
 
-# Normalization Mode - see src/training/language_specific.sh for details. Default: $(NORM_MODE)
-NORM_MODE = 2
+# Finetune Training Type - Impact, Plus, Layer
+FINETUNE_TYPE =
+
+# Normalization mode for unicharset_extractor and Pass through Recoder for combine_lang_model
+ifeq ($(LANG_TYPE),Indic)
```

Comment:
Document that `$LANG_TYPE` has an influence here. Maybe add

```make
# bla
LANG_TYPE ?=
```

which won't set LANG_TYPE but still list it in the variables section of the help target.

Reply:
OK. Thanks!